packages/odata-service-inquirer/test/unit/prompts/sap-system/questions.test.ts

```diff
@@ -1,10 +1,19 @@
 import { initI18nOdataServiceInquirer } from '../../../../src/i18n';
 import { getNewSystemQuestions } from '../../../../src/prompts/datasources/sap-system/new-system/questions';
+import { getHostEnvironment, hostEnvironment } from '@sap-ux/fiori-generator-shared';
+
+jest.mock('@sap-ux/fiori-generator-shared', () => ({
```

Comment:
Curious as to why this is necessary now, we provide a mock when the related code was internal to this module and the behaviour should not have changed?

Reply:
It must have failed at some stage while I was updating and I added this, but it was unnecessary and should have been removed. 
Thanks

Reply:
It was passing locally but failing again on when I pushed here. 
CLI prompt is not returned without the mock.. I'll check

Reply:
OK, I thought we would still pass the `isYUI` optional arg and use that rather than determining from the process since the allows consumers (and tests) to set the behaviour. The fallback would be to use the process...